server/repository/migrations/postgres/2022-01-27T08-00_create_changelog_table/up.sql

```diff
@@ -18,47 +18,52 @@ CREATE TYPE row_action_type AS ENUM (
 );
 
 CREATE TABLE changelog (
-    id BIGSERIAL,
+    cursor BIGSERIAL,
     -- the table name where the change happend
     table_name changelog_table_name NOT NULL,
     -- row id of the modified row
-    row_id TEXT NOT NULL,
+    record_id TEXT NOT NULL,
     -- Sqlite only fires INSERT when doing an upsert (it does a delete + insert) for this reason
     -- use UPSERT.
-    row_action row_action_type NOT NULL
+    row_action row_action_type NOT NULL,
+    -- TODO comment
```

Comment:
TODO?

Reply:
fb8a00dd85df6b719b2d52503a4145c3198939b1 not super happy with explanation, but wanted to be brief and avoid duplication (to reduce the chance of comment becoming stale)

Reply:
Does it refer to the SQLite comment?
How about...
   -- SQLite only fires INSERT triggers when doing an upsert (it does a delete + insert) for this reason row_action is always recorded as UPSERT for both INSERT and UPSERT statements.

Reply:
Ohh, this was about the TODO comment to explain what  name_id and store_id are in changelog, the sqlite comment was removed